src/allmydata/web/root.py

```diff
@@ -182,9 +183,44 @@ def data_import_path(self, ctx, data):
     def render_my_nodeid(self, ctx, data):
         tubid_s = "TubID: "+self.client.get_long_tubid()
         return T.td(title=tubid_s)[self.client.get_long_nodeid()]
+
     def data_my_nickname(self, ctx, data):
         return self.client.nickname
 
+    def renderHTTP(self, ctx):
+        req = inevow.IRequest(ctx)
+        t = get_arg(req, "t")
```

Comment:
I'm happy with making a rule that new code should use explicit prefixes, and eventually we can convert everything to that format (making gratuitous conversions right now is probably a bad idea, given the large+old branches that we're hoping to land soon).